### PR TITLE
Implementation of normalization

### PR DESCRIPTION
**Pull request by YashitaBansal**
Input:
`std::vector<double> values={1,2,3,4,5};`

Output:
[0,0.25,0.5,0.75,1]

### CODE REVIEW

**Review by ken1000minus7 — APPROVED** [review]

File: src/slowmokit/methods/preprocessing/normalization.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/**
+ * @file methods/preprocessing/normalization.CPP
+ *
+ * Implementation of the Accuracy score main program
+ */
+#include "normalization.hpp"
+template <class T>
+void normalize(std::vector<T> &values)
+{
+    double minimum = *min_element(values.begin(), values.end());
```
Comment (uttammittal02) [inline src/slowmokit/methods/preprocessing/normalization.cpp:10]: @YashitaBansal Replace double with T here as the datatype of min/max value will be the same as that in values vector

**Review by uttammittal02 — CHANGES_REQUESTED** [review]

File: src/slowmokit/methods/preprocessing/standardization.cpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        std::cerr << "Warning: Standard Deviation is zero" << std::endl;
+        return;
+    }
+    for (int i = 0; i < n; i++)
+    {
+        values[i] = (values[i] - mean) / stdDev;
+    }
+}
+
+int main(){
```
Comment (uttammittal02) [inline src/slowmokit/methods/preprocessing/standardization.cpp:36]: Main Function isn't needed here. Kindly remove it from here.

**Review by uttammittal02 — COMMENTED** [review]
Do the required changes asap

**Review by uttammittal02 — APPROVED** [review]
Nice

**Pull request merged**